remote-query/remote-query-server/src/test/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptorTest.java

```diff
@@ -24,7 +24,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-@Test(groups = "functional", testName = "query.remote.ProtobufMetadataManagerInterceptorTest")
+@Test(groups = "functional", testName = "query.remote.impl.ProtobufMetadataManagerInterceptorTest")
```

Comment:
TestNG test names don't need to match the package name exactly, I'd rather leave out the `impl` part.